src/Esfa.Vacancy.Api.Types/Vacancy.cs

```diff
@@ -13,7 +13,7 @@ public class Vacancy
         /// <value>
         /// The reference.
         /// </value>
-        public int Reference { get; set; }
+        public int VacancyReference { get; set; }
```

Comment:
We should probably change the name of the parameter on the VacancyController as well. At the moment it is 'id' but this does not make sense and it is what is also shown on the swagger page 😒 

But changing this field name is cool.